### DISCUSSION TITLE
Capitalization options for the User component

### DISCUSSION THREAD

**Discussion by gregorgabrovsek** [heroui-inc/heroui#665] [Feedback]
Hey,

when using the `User` component, I noticed that every word of the string given as the `name` prop is automatically capitalized. It appears that [this](https://github.com/nextui-org/nextui/blob/main/packages/react/src/user/user.styles.ts#L39) line is responsible for the described behavior.

I'm thinking that it would be a good idea to add a prop to the `User` component, which would allow users to modify this behavior (of course, the default capitalization option could be the current behavior - just to avoid breaking changes).

There are two benefits I have in mind:
- If the `User` component is used by the developer for displaying actual names/people: this would allow the component to display the names more correctly, as [not all names have all words capitalized](https://en.wikipedia.org/wiki/Anthony_van_Dyck).
- In case the developer uses the `User` component for something else entirely: capitalizing all words can be a nuisance

I hope this makes sense :)

**Comment by jrgarciadev** [comment]
Hey @GoesUp thanks for your suggestion, I just sent a commit removing the text transform, it will be available in the `v11`
https://github.com/nextui-org/nextui/commit/30b22cb2b8422c5df82dc609e1a3fb8fe1f79eaf
